flask_annex/file.py

```diff
@@ -95,11 +95,12 @@ def _ensure_key_dir(self, key):
         os.makedirs(dir_name, exist_ok=True)
 
     def send_file(self, key):
+        if flask.__version__ >= "2.2.0":
```

Comment:
From https://flask.palletsprojects.com/en/2.2.x/changes/#version-2-2-0

@sloria was this what you meant?

Reply:
nitpicky, but it's generally good to avoid comparing strings for version checks, since this would break, for example with `flask.__version__ = "10.0.0"`.

better to use `packaging.version.Version`

```suggestion
        if Version(flask.__version__) >= Version("2.2.0"):
```

then up top

```python
from packaging.version import Version
```

you'll need to add `packaging>=17.0` to `install_requires` in `setup.py`

Reply:
No worries, makes sense. Updated in latest.